spec/features/past_orders_page_spec.rb

```diff
@@ -59,26 +59,6 @@
       end
     end
 
-    context "five hours after order is made" do
-      it "should have an order status of shiped" do
-        order.created_at = Time.now - 5.hour
-        order.save
-        signin_helper(order.user.email, order.user.password)
-        visit past_orders_path
-        expect(page).to have_content("Shipped")
-      end
-    end
-
-    context "14 hours after order is made" do
-      it "should have an order status of pending" do
-        order.created_at = Time.now - 13.hour
-        order.save
-        signin_helper(order.user.email, order.user.password)
-        visit past_orders_path
-        expect(page).to have_content("Delivered")
-      end
-    end
-
```

Comment:
Hola Godwin,
Good work fixing this feature test. 
Please resolve this on the Poro class test also. Remove the static tests on the PORO class as well.